ui.R

```diff
@@ -104,41 +129,37 @@ dashboardPage(
                 "Filter on number of free throws attempted:",
                 min = 0,
                 max = 881,
+                step = 5,
                 value = c(0, 881)
               )
             ),
             img(src = "Giannis.png", height = 219, width = 300, align = "middle")
           ),
-
+          
           # Column two displays the Histogram of the distrubition of the free throw attempts
           column(
             8,
-            h1("Histogram"),
-
+            h2("Histogram"),
+            
             plotOutput("histogramNBA"),
             # Add rollover for Histogram of Free Throw Proportion Plot
             bsPopover(id = "histogramNBA", title = " ", content = "This histogram filters the NBA players based on games played or free throw attempts. Changing the slider will adjust the number of players that fit the selected criteria.", placement = "left", trigger = "hover", options = NULL)
           ),
-
+          
           # A box with information to get students thinking and transitioning into part 2
-          box(width = 12, background = "blue", h4("Try to think about what the median and mean of FT% are and what range you might expect most of the players to fall in. "))
+          box(width = 12, background = "purple", h4("Try to think about what the median and mean of FT% are and what range you might expect most of the players to fall in. "))
```

Comment:
Remove the box. Move the text into the paragraph that will have the popover text.